### ISSUE TITLE
showing on more content (link) is in between stream objects

### ISSUE THREAD

**Issue by 001101** [humhub/humhub#2436]
hello,

the small notice about one more element was just now between some stream objects, the content type  was link.

greets

**Comment by buddh4** [comment]
Is this a question or don't i understand the issue? ;) This feature should reduce for example the spam of tasks, files etc. Perhaps we can make it configurable in a future version.

**Comment by 001101** [comment]
For a Community it is a bug and no feature, sorry, but this notification is not big enough to get noticed, it is not usable and should be a option, but it is no feature.

**Comment by buddh4** [comment]
You mean its a problem for you in case only one content is missing or the overall functionality. I'am happy not to see each stream entry for 20 uploaded files or 12 tasks etc. Currently you can disable this behaviour vor specific content types by the following `protected/config/common.php`:

```
return [
    'modules' => [
        'stream' => [
            'streamSuppressQueryIgnore' => [
                'humhub\modules\tasks\models\Task'
            ]
        ]
    ]
];
```

**Comment by 001101** [comment]
Which I can fully understand, but in my case, I running a small community site and when someone enters a event in the calendar and it got hidden, it is not useful for making new friends :)

So I have to do this behaviour for any module which got hidden?

**Comment by buddh4** [comment]
Currently, yes, we can introduce a single configuration for disabling the behaviour in the next release.

**Comment by 001101** [comment]
Now I have had time to get more in touch with the behavior, but I could not solve my problem.
I have about 40 default spaces in my community and every time a new user signs up, the stream get flooded with the messages that user XY joined all those spaces.

Now I have added this to protected/config/common.php:

`return [
    'modules' => [
        'stream' => [
            'streamSuppressQueryIgnore' => [
                'humhub\modules\space\models\Membership'
            ]
        ]
    ]
];`

But the issue is still alive, which module and model is the cause for that behavior?

thanks in advice!

**Comment by buddh4** [comment]
You mean the activity stream?

I think its on the secondary feature list of v1.3 to be able to prevent such activities or make it a bit more configurable. I'am currently not sure if the suppression logic even works with the activity stream.

**Comment by 001101** [comment]
Yes I mean the activity widget in this case, is there maybe another possibility for merge such entries?

**Comment by buddh4** [comment]
Currently not, as I said we are aware of the issue, hopefully we'll find a solution for this soon.

**Comment by 001101** [comment]
Great, good to hear :)
